core/web/jobs_controller.go

```diff
@@ -113,7 +113,7 @@ func (jc *JobsController) Create(c *gin.Context) {
 			return
 		}
 	case job.OffchainReporting2:
-		jb, err = offchainreporting2.ValidatedOracleSpecToml(jc.App.GetChainSet(), request.TOML)
+		jb, err = offchainreporting2.ValidatedOracleSpecToml(jc.App.GetConfig(), request.TOML)
```

Comment:
This is not correct because it ignores chain-specific OCR config settings. You must use the chainset config here for the given spec

Reply:
I removed the chain specific OCR settings for OCR2 from the env vars to help simplify this upgrade path. My plan is:
1. Use OCR2 for solana exclusively, support global OCR2 env vars or job specificed OCR2 vars
2. Figure out a way to extend our chain specific config to non-evm chains
3. Support ethereum jobs on OCR2